proto/types/sentry/sentry.proto

```diff
@@ -6,25 +6,25 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 import "proto/types/commonpb/v3/common.proto";
 
 enum BootstrapInfraType {
-  ClusterBootstrapInfra = 0;
```

Comment:
There are a few things like this where `0` is not the `UNSPECIFIED` / `NOT_SET` value because we don't have a not set state. We will have  to handle them differently (either add extra code to use default values when not set [this will then have two values in enum] or silence the specific lint warning). Changing `ClusterBootstrapInfra` to `NOT_SET` blindly will change the meaning.

Reply:
I agree, we may have to revisit these. A few including this is not used anywhere, let's remove those altogether.